core/app/flags/flags.go

```diff
@@ -176,7 +176,15 @@ func (s *Set) Bind(name string, value interface{}, help string) {
 		default:
 			fullname = name + "-" + fname
 		}
-		s.Bind(fullname, field.Addr().Interface(), usage)
+
+		if tags.Get("repeated") == "true" {
```

Comment:
Do we need the `repeated` tag? Isn't having the type a slice enough info?

Reply:
Done. Changed it so that slices are automatically repeated flags.